lib/src/folding/parsers/highlight.dart

```diff
@@ -66,12 +66,13 @@ class HighlightFoldableBlockParser extends TextFoldableBlockParser {
       return;
     }
 
-    setFoundImportTerminator();
-    submitCurrentLine();
-
     startBlock(lineIndex, FoldableBlockType.multilineComment);
 
-    addToLineIndex(newlineCount);
+    for (int i = 0; i < newlineCount; i++) {
+      submitCurrentLine();
+      addToLineIndex(1);
+    }
```

Comment:
It is needed to preserve line semantics for each line in between start and end of a multiline comment block, so that the line sequence in `_linesWithSemantics` will not be interrupted.

Reply:
Add `_foundMultiLineComment`, add a multiline comment semantic.